src/Kros.ProblemDetails.Extensions/ProblemDetailsExtensions.cs

```diff
@@ -0,0 +1,48 @@
+﻿using FluentValidation;
+using Hellang.Middleware.ProblemDetails;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using System;
+
+namespace Kros.ProblemDetails.Extensions
+{
+    /// <summary>
+    /// Hellang problem details service extensions.
+    /// </summary>
+    public static class ProblemDetailsExtensions
+    {
+        /// <summary>
+        /// Registers Hellang problem details to IoC container.
+        /// </summary>
+        /// <param name="services">IoC container.</param>
+        /// <param name="environment">Current environment.</param>
+        /// <param name="configAction">Additional action for configuring Hellang problem details.</param>
+        public static IServiceCollection AddCustomProblemDetails(
+            this IServiceCollection services,
+            IWebHostEnvironment environment,
+            Action<ProblemDetailsOptions> configAction = null)
+            =>  services.AddProblemDetails(p =>
+            {
+                p.IncludeExceptionDetails = (context, ex) => IncludeExceptionDetails(environment, ex);
+                p.SourceCodeLineCount = 0;
```

Comment:
Toto čo znamená?

Reply:
IncludeExceptionDetails nastavuje, ci sa maju pri chybe vratit podrobnejsie detaily - error message, podrobny call stack atd...
To som zapol iba pre test a devel prostredie a zaroven vypol pri fluent validacnej chybe.
SourceCodeLineCount urcuje kolko riadkov kodu pred a po chybovom riadku zobrazovat pri podrobnejsich detailoch. To mi prislo uz prehnane, aby sa nam zobrazol kod v chybe, tak som nastavil 0 riadkov.

Reply:
No v dev prostredí mi to príde fajn, že rovno vidím kód. @satano čo si myslíš ty?

Reply:
1. Mne to príde celkom fajn vidieť aj kúsok zdrojáku, keď platí podmienka `IncludeExceptionDetails`.
2. To číslo sa používa iba v prípade, že sú žiadane detaily výnimky (`IncludeExceptionDetails == true`). Takže je bezpečné ho nechať na pokoji a v prípade že chceme aj detaily, tak bude aj kúsok zdrojáku.

Reply:
Upravil som, otazne je este test prostredie. Tam sme doteraz call stack aj text chyby mali, bolo by fajn ho tam ponechat, bude to uzitocne. Ale tym padom sa bude aj tam zobrazovat cast kodu.
Zatial som to ponechal pre dev aj test. Co si o tom myslite? @satano @Burgyn

Reply:
Ja som s tým v pohode